setup.py

```diff
@@ -32,35 +32,35 @@
 @author: Andy Georges (Ghent University)
 @author: Kenneth Hoste (Ghent University)
 """
+
+# vsc-base setup.py needs vsc.install, which is currently shipped as part of vsc-base
+# vsc.install doesn't require vsc-base, so we could move it to it's own repo and only 
+# have this hack in the setup.py of vsc.install (and set it as build_requires)
+# until then...
 import os
 import sys
-
 sys.path.insert(0, os.path.join(os.path.dirname(os.path.abspath(__file__)), "lib"))
 
+
 import vsc.install.shared_setup as shared_setup
-from vsc.install.shared_setup import ag, kh, jt, sdw
+from vsc.install.shared_setup import ag, kh, jt, sdw, URL_GH_HPCUGENT
 
 def remove_bdist_rpm_source_file():
     """List of files to remove from the (source) RPM."""
     return []
 
 shared_setup.remove_extra_bdist_rpm_files = remove_bdist_rpm_source_file
-shared_setup.SHARED_TARGET.update({
-    'url': 'https://github.com/hpcugent/vsc-base',
-    'download_url': 'https://github.com/hpcugent/vsc-base',
-    'zip_safe': True,
-})
 
 
 PACKAGE = {
     'name': 'vsc-base',
-    'version': '2.3.0',
     'author': [sdw, jt, ag, kh],
     'maintainer': [sdw, jt, ag, kh],
     'packages': ['vsc', 'vsc.install', 'vsc.utils'],
     'scripts': ['bin/logdaemon.py', 'bin/startlogdaemon.sh', 'bin/bdist_rpm.sh', 'bin/optcomplete.bash'],
     'install_requires' : ['setuptools'],
+    'zip_safe': True,
```

Comment:
@ageorges: shouldn't matter whether vsc-utils touches `vsc/utils` too, vsc-base by itself works just fine when installed as a zipped egg

However, other stuff that requires it as a dep doesn't support using it when it's installed as a zipped egg yet *cough* EasyBuild *cough*

So, for now, Jenkins shouldn't install it as a zipped egg after a successful merge/test, i.e. it should use `easy_install -U`